Client/Frontend/Browser/URLBarView.swift

```diff
@@ -148,6 +133,26 @@ class URLBarView: UIView, BrowserLocationViewDelegate, AutocompleteTextFieldDele
         finishEditingAnimation(false)
     }
 
+    private func makeTabsButton(count: Int) -> UIButton {
+        let tabsButton = InsetButton()
+        tabsButton.setTranslatesAutoresizingMaskIntoConstraints(false)
```

Comment:
Doesn't SnapKit do this already? Or is this a special case where we do need to call it?